lib/releaselib.sh

```diff
@@ -631,6 +631,8 @@ release::gcs::locally_stage_release_artifacts() {
       || return 1
     release::gcs::stage_and_hash $gcs_stage $gci_path/configure.sh extra/gce \
       || return 1
+    release::gcs::stage_and_hash $gcs_stage $gci_path/shutdown.sh extra/gce \
```

Comment:
won't this cause problems on older branches where this file doesn't exist?

Reply:
Is there a versioning story for this lib? How do you recommend I add this extra dependency?

Reply:
I don't think there is one, but maybe add a conditional to see if the file exists first? see e.g. L613-618

Reply:
L620-624 is another good example

Reply:
@ixdy updated code. PTAL